filebeat/harvester/reader/multiline.go

```diff
@@ -156,37 +158,37 @@ func (mlr *Multiline) readNext() (Message, error) {
 			}
 
 			// handle error without any bytes returned from reader
-			if message.Bytes == 0 {
-				// no lines buffered -> return error
-				if mlr.numLines == 0 {
-					return Message{}, err
-				}
-
-				// lines buffered, return multiline and error on next read
-				m := mlr.finalize()
-				mlr.err = err
-				mlr.setState((*Multiline).readFailed)
-				return m, nil
-			}
+			//if message.Bytes == 0 {
```

Comment:
is there a chance a reader can return a valid message + some error? If yes, the message must be handled.